source/Sylvan.Data.Excel/Xls/XlsWorkbookReader.cs

```diff
@@ -788,6 +818,31 @@ public override bool IsDBNull(int ordinal)
 			return ref row[dataIdx];
 		}
 
+
+		string? GetHeaderString(int ordinal)
```

Comment:
string is already nullabe because of the reference type, the `?` is pointless here.

Reply:
That was true prior to C# 8, and in fact, it would have been illegal syntax. However, C# 8 added the  nullable reference types features, which is what this is. It advertises that the return value may be null.
https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references